src/component/common/ProtectedRoute/ProtectedRoute.jsx

```diff
@@ -1,26 +0,0 @@
-import { Route, useLocation, Redirect } from 'react-router-dom';
-
-const ProtectedRoute = ({
-    component: Component,
-    unauthorized,
-    renderProps = {},
-    ...rest
-}) => {
-    const { pathname } = useLocation();
-    const loginLink =
-        pathname.length > 1 ? `/login?redirect=${pathname}` : '/login';
```

Comment:
@FredrikOseberg shouldn't `loginLink` here be wrapped somehow? `getBasePath`?

Reply:
In this case I think the redirect component from react router takes care of the sub path since it's set in the router as base name. But great question! Feel free to verify this too.